README.md

```diff
@@ -6,6 +6,54 @@
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/great-expectations/cloud/main.svg)](https://results.pre-commit.ci/latest/github/great-expectations/cloud/main)
 [![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/charliermarsh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
 
+## Quick Start
+
+### Python
+
+#### Install
+```console
+pip install great_expectations_cloud
+```
+##### Optional Dependencies
+```console
+pip install 'great_expectations_cloud[sql]'
+```
+
+```console
+$ gx-agent --help
+usage: gx-agent [-h] [--log-level LOG_LEVEL] [--skip-log-file SKIP_LOG_FILE] [--log-cfg-file LOG_CFG_FILE] [--version]
```

Comment:
We need to use more lazy imports or something, showing this help page is unbearable slow.
On my machine it takes like 5+ seconds